src/components/AddForm.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react'
+
+export default function AddForm (props){
+  return(
+    <form onSubmit={props.addFood}>
```

Comment:
working with a simple form and passing the information through the event is fine. But in react you can use the state to dinamically change the values of the input field. And then use the information in the state to use it on the function call via parameter.